com.unity.multiplayer.mlapi/Prototyping/NetworkTransform.cs

```diff
@@ -283,19 +283,14 @@ private void SubmitTransformServerRpc(Vector3 position, Vector3 eulerAngles, Ser
             {
                 for (int i = 0; i < NetworkManager.ConnectedClientsList.Count; i++)
                 {
-                    if (!m_ClientSendInfo.ContainsKey(NetworkManager.ConnectedClientsList[i].ClientId))
+                    if (!m_ClientSendInfo.TryGetValue(NetworkManager.ConnectedClientsList[i].ClientId, out ClientSendInfo info))
                     {
-                        m_ClientSendInfo.Add(NetworkManager.ConnectedClientsList[i].ClientId, new ClientSendInfo()
-                        {
-                            LastMissedPosition = null,
-                            LastMissedRotation = null,
-                            LastSent = 0
-                        });
+                        info = new ClientSendInfo() { LastMissedPosition = null, LastMissedRotation = null, LastSent = 0 };
+                        m_ClientSendInfo.Add(NetworkManager.Singleton.ConnectedClientsList[i].ClientId, info);
```

Comment:
While you're optimizing lookups, does it make sense to cache the indexed connected client?